spec/formats/configuration_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe "Format configuration" do
+  supertypes = YAML.load_file("app/formats/supertypes.yml")
+  document_types = YAML.load_file("app/formats/document_types.yml")
+
+  describe "Supertypes" do
```

Comment:
Could we collapse these two describes into one to make it a bit more readable?

Reply:
(Or the inner describe and the it - it's very block-tastic at the moment ;-).

Reply:
Good point, I merged the two `describes`!